src/test/java/org/opensearch/flowframework/rest/RestWorkflowStateTenantAwareIT.java

```diff
@@ -158,6 +165,166 @@ public void testWorkflowStateCRUD() throws Exception {
             assertEquals("COMPLETED", stateMap.get("state"));
         }, 20, TimeUnit.SECONDS);
 
+        // Verify created resources and ml client calls for connector, model & agent
+        response = makeRequest(tenantRequest, GET, WORKFLOW_PATH + workflowId + STATUS_ALL);
+        assertOK(response);
+        Map<String, Object> statemap = responseToMap(response);
+        List<Map<String, Object>> resourcesCreated = (List<Map<String, Object>>) statemap.get("resources_created");
+        String connectorId = resourcesCreated.get(0).get("resource_id").toString();
+        assertNotNull(connectorId);
+
+        String modelId = resourcesCreated.get(1).get("resource_id").toString();
+        assertNotNull(modelId);
+
+        String modelGroupId = resourcesCreated.get(2).get("resource_id").toString();
+        assertNotNull(modelGroupId);
```

Comment:
I think the model group ID matches the model ID here (or at least contains it in the string).